easybuild/easyconfigs/l/LAMMPS/LAMMPS-3Mar2020-intel-2020a-Python-3.8.2-kokkos.eb

```diff
@@ -0,0 +1,107 @@
+checksums = [
+    'a1a2e3e763ef5baecea258732518d75775639db26e60af1634ab385ed89224d1',  # stable_3Mar2020.tar.gz
+    '5e5211f64113e8a38c102f8542fd59169b689fdb92afbff29c3b88f802f1a96c',  # lammps_vs_yaff_test_single_point_energy.py
+    '7f010853d81022f286cf32e3babe252d5cc7c0bfb274bee5a2c64e810e170239',  # LAMMPS-3Mar2020_fix-docs-build.patch
+]
+
+local_source_dir_name = '%(namelower)s-%(version)s'
+
+builddependencies = [
+    ('CMake', '3.16.4'),
+    ('pkg-config', '0.29.2'),
+    ('archspec', '0.1.0', local_python_versionsuffix),
+]
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.4'),
+    ('netCDF', '4.7.4'),
+    ('GSL', '2.6'),
+    ('zlib', '1.2.11'),
+    ('gzip', '1.10'),
+    ('cURL', '7.69.1'),
+    ('HDF5', '1.10.6'),
+    ('tbb', '2020.1'),
+    ('PCRE', '8.44'),
+    ('libxml2', '2.9.10'),
+    ('FFmpeg', '4.2.2'),
+    ('Voro++', '0.4.6'),
+    ('kim-api', '2.1.3'),
+    ('Eigen', '3.3.7'),
+    ('yaff', '1.6.0', local_python_versionsuffix),
+    ('PLUMED', '2.6.0', local_python_versionsuffix),
+    ('ScaFaCoS', '1.0.1'),
+    # See below for why this is not included
+    # ('VTK', '8.2.0', local_python_versionsuffix),
+]
+
+# To use additional custom configuration options, use the 'configopts' easyconfig parameter
+# See docs and lammps easyblock for more information.
+# https://github.com/lammps/lammps/blob/master/cmake/README.md#lammps-configuration-options
+# configopts = " "
+
+# auto-enabled by easyblock
+# 'GPU'    - if cuda package is present and kokkos is disabled
+# 'KOKKOS' - if kokkos is enabled (by default)
+#
+# not enabled (yet), needs more work/additional dependencies:
+# 'LATTE', - https://lammps.sandia.gov/doc/Build_extras.html#latte-package
+# 'MSCG',  - https://lammps.sandia.gov/doc/Build_extras.html#mscg-package
+general_packages = [
+    'ASPHERE', 'BODY', 'CLASS2', 'COLLOID', 'COMPRESS', 'CORESHELL', 'DIPOLE',
+    'GRANULAR', 'KIM', 'KSPACE', 'MANYBODY', 'MC', 'MESSAGE', 'MISC',
+    'MOLECULE', 'MPIIO', 'PERI', 'POEMS', 'PYTHON', 'QEQ', 'REPLICA', 'RIGID',
+    'SHOCK', 'SNAP', 'SPIN', 'SRD', 'VORONOI',
+]
+
+# not enabled (yet), needs more work/additional dependencies:
+# ADIOS - https://lammps.sandia.gov/doc/Build_extras.html#user-adios-package
+# AWPMD - https://lammps.sandia.gov/doc/Build_extras.html#user-awpmd-package
+# QUIP  - https://lammps.sandia.gov/doc/Build_extras.html#user-quip-package
+# VTK - support is available in the foss version but currently fails to build for intel
+# due to https://software.intel.com/en-us/forums/intel-fortran-compiler/topic/746611
+# see https://github.com/lammps/lammps/issues/1964 for details
+user_packages = [
+    'ATC', 'BOCS', 'CGDNA', 'CGSDK', 'COLVARS', 'DIFFRACTION', 'DPD', 'DRUDE',
+    'EFF', 'FEP', 'H5MD', 'LB', 'MANIFOLD', 'MEAMC', 'MESO', 'MGPT', 'MISC',
+    'MOFFF', 'MOLFILE', 'NETCDF', 'PHONON', 'PLUMED', 'PTM', 'QTB', 'REAXC',
+    'SCAFACOS', 'SMD', 'SMTBQ', 'SPH', 'TALLY', 'UEF', 'YAFF'
+]
+
```

Comment:
```suggestion

# Initialize MPI for the build, otherwise any executables linking to MPI will crash (e.g. sphinx-build)
# This is known issue with intel/2020a. See easybuilders/easybuild-easyconfigs issue #10213
prebuildopts = "%(mpi_cmd_prefix)s"

```

Reply:
There was an alternative fix for this, with a patch (which also avoided that stuff is auto-downloaded by `pip install` to build the docs)... see #10191

Reply:
The patch for the docs is already included in this PR. The problem is that `sphinx-build` somehow links to MPI and in `intel/2020a` this means needing to run with `%(mpi_cmd_prefix)s`. Second problem is that adding the template for `%(mpi_cmd_prefix)s` cannot be patched.
I agree that using `prebuildopts = "%(mpi_cmd_prefix)s"` might be overkill though. I made a more specific fix for this issue in my review below.